sdk/stdlib-sys/src/intrinsics/mod.rs

```diff
@@ -1,5 +1,6 @@
 use core::ops::{Deref, DerefMut};
 
+pub mod debug;
```

Comment:
I'm not sure that I understand why it's public.

Reply:
Unlike the other two modules, which we're wildcard re-exporting the contents of from the root, we only need to export the `debug` module and require fully-qualified paths instead to access its declarations.